### DISCUSSION TITLE
Cannot get GB/GBCs to dump successfully (checksum error)

### DISCUSSION THREAD

**Discussion by staze** [sanni/cartreader#99] [Need help]
Have double checked switches. Can dump GBA just fine. For whatever reason, I can't get GB/GBC to dump successfully. Have tested only with MBC1 (Link's awakening US, Tetris DX, and Super Mario Land). Don't have any MBC3 or MBC5 carts. 

Thanks!

**Comment by sanni** [comment]
Does dumping N64 and SNES work?

**Reply by staze** [reply]
Yup. N64 on 3v, SNES on 5v. Seem to work fine. I will admit, I haven't tested the N64 dumps, but the SNES ones work great.

**Comment by sanni** [comment]
Try cleaning the GB carts contacts with a q-tip and rubbing alcohol until the q-tip doesn't turn black anymore. Of the games you mentioned I only have Super Mario Land but it dumps fine for me. I know that the GameBoy code currently reads a little too fast which mainly affects slower cartridges like the GameBoy camera but it might ofc prevent other carts from dumping correctly too.

**Reply by staze** [reply]
Have cleaned. same result. so yeah, maybe it's just reading too fast for the older carts. no worries. the SNES carts are what I built my Sanni for, and that seems to work great. Thanks!
